### DISCUSSION TITLE
Can't login to admin after clean install

### DISCUSSION THREAD

**Discussion by s22-tech** [LycheeOrg/Lychee#2491] [Q&A] [answered]
I just installed 5.5.0 and I can't login to the backend.  When I click the little icon in the top left corner, nothing happens (see attached photo).

These lines are displayed in the console:

```
Error with Permissions-Policy header: Unrecognized feature: 'ambient-light-sensor'.
Error with Permissions-Policy header: Unrecognized feature: 'battery'.
Error with Permissions-Policy header: Unrecognized feature: 'document-domain'.
Error with Permissions-Policy header: Unrecognized feature: 'execution-while-not-rendered'.
Error with Permissions-Policy header: Unrecognized feature: 'execution-while-out-of-viewport'.
Error with Permissions-Policy header: Unrecognized feature: 'navigation-override'.
Error with Permissions-Policy header: Unrecognized feature: 'web-share'.

GET https://domain.com/livewire/livewire.js?id=87e1046f net::ERR_ABORTED 404 (Not Found)   gallery:587 


Refused to execute script from 'https://domain.com/livewire/livewire.js?id=87e1046f' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
app-D721H7k1.js:20    gallery:1 

Uncaught ReferenceError: Alpine is not defined
    at app-D721H7k1.js:20:1020
(anonymous) @ app-D721H7k1.js:20
```
EDIT: For some reason, my screenshot refuses to upload here.

**Comment by ildyria** [comment]
> Refused to execute script from 'https://domain.com/livewire/livewire.js?id=87e1046f' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.
app-D721H7k1.js:20    gallery:1 
> 
> Uncaught ReferenceError: Alpine is not defined
    at app-D721H7k1.js:20:1020
(anonymous) @ app-D721H7k1.js:20

WTF.

**Reply by ildyria** [reply, marked as answer]
Can you go into your `.env` and set `LIVEWIRE_ENABLED=false`

**Reply by s22-tech** [reply]
That was it.  Thank you.

**Reply by ildyria** [reply]
It is not a final solution, but more of a temporary one.
Now can you go to your diagnostics and see what is in there?

**Comment by s22-tech** [comment]
Diagnostics
    -------
    Warning: /home/*************************************/storage/image-jobs/ has permissions 0775, but should have 2775
    Warning: /home/*************************************/storage/livewire-tmp/ has permissions 0775, but should have 2775
    Warning: Dropbox import not working. dropbox_key is empty.
    Warning: You may experience problems when uploading a photo of large size. Take a look in the FAQ for details.
    Warning: You may experience problems when uploading a photo of large size. Take a look in the FAQ for details.
    Warning: You may experience problems when uploading a photo of large size or handling many/large albums. Take a look in the FAQ for details.
    Warning: Pictures that are rotated lose their metadata! Please install Imagick to avoid that.
    Warning: zend.assertions is disabled although Lychee is in debug mode. For easier debugging code generation for assertions should be enabled.
    Error: APP_URL (http://*************ys.com) does not match the current url (https://*************ys.com).
          This will break WebAuthn authentication.
    Error: APP_URL (http://*************ys.com) does not match the current url (https://*************ys.com).
          This will prevent images from being properly displayed.


    System Information
    --------------
    Lychee Version (git):                    master (dcf54f0) -- Up to date (-1.265982158217E-5 years ago).
    DB Version:                              5.5.0
    
    composer install:                        --no-dev
    APP_ENV:                                 production
    APP_DEBUG:                               true
    APP_URL:                                 set
    APP_DIR:                                 default
    LOG_VIEWER_ENABLED:                      true
    LIVEWIRE_ENABLED:                        false
    
    System:                                  Linux
    PHP Version:                             8.3.8
    PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
    Timezone:                                Europe/Rome
    Max uploaded file size:                  10M
    Max post size:                           8M
    Livewire chunk size:                     8.00 MB
    Max execution time:                      30
    MySQL Version:                           10.6.18-MariaDB
    
    exec() Available:                        yes
    Imagick Available:                       -
    Imagick Enabled:                         1
    Imagick Version:                         -
    GD Version:                              bundled (2.1.0 compatible)
    Number of foreign key:                   116 found.

Request space usage <>

    Config Information
    --------------
    version:                                 050500
    check_for_updates:                       0
    sorting_photos_col:                      taken_at
    sorting_photos_order:                    ASC
    sorting_albums_col:                      max_taken_at
    sorting_albums_order:                    ASC
    imagick:                                 1
    skip_duplicates:                         0
    small_max_width:                         0
    small_max_height:                        360
    medium_max_width:                        1920
    medium_max_height:                       1080
    lang:                                    en
    image_overlay_type:                      desc
    default_license:                         none
    compression_quality:                     90
    grants_full_photo_access:                1
    delete_imported:                         0
    mod_frame_enabled:                       1
    mod_frame_refresh:                       30
    thumb_2x:                                1
    small_2x:                                1
    medium_2x:                               1
    landing_page_enable:                     0
    force_migration_in_production:           0
    search_public:                           0
    hide_version_number:                     0
    SL_enable:                               0
    SL_for_admin:                            0
    recent_age:                              1
    grants_download:                         0
    photos_wraparound:                       1
    map_display:                             0
    zip64:                                   1
    map_display_public:                      0
    map_provider:                            Wikimedia
    force_32bit_ids:                         0
    map_include_subalbums:                   0
    update_check_every_days:                 3
    has_exiftool:                            0
    share_button_visible:                    0
    import_via_symlink:                      0
    has_ffmpeg:                              0
    location_decoding:                       0
    location_decoding_timeout:               30
    location_show:                           1
    location_show_public:                    0
    rss_enable:                              0
    rss_recent_days:                         7
    rss_max_items:                           100
    prefer_available_xmp_metadata:           0
    editor_enabled:                          1
    lossless_optimization:                   0
    swipe_tolerance_x:                       150
    swipe_tolerance_y:                       250
    local_takestamp_video_formats:           .avi|.mov
    log_max_num_line:                        1000
    unlock_password_photos_with_url_param:   0
    nsfw_visible:                            0
    nsfw_blur:                               0
    nsfw_warning:                            0
    nsfw_warning_admin:                      0
    nsfw_banner_override:                    
    map_display_direction:                   1
    album_subtitle_type:                     oldstyle
    upload_processing_limit:                 4
    new_photos_notification:                 0
    legacy_id_redirection:                   1
    zip_deflate_level:                       6
    default_album_protection:                1
    album_decoration:                        layers
    album_decoration_orientation:            row
    allow_username_change:                   1
    auto_fix_orientation:                    1
    use_job_queues:                          0
    random_album_id:                         starred
    use_last_modified_date_when_no_exif_date: 0
    layout:                                  justified
    date_format_photo_thumb:                 M j, Y, g:i:s A e
    date_format_photo_overlay:               M j, Y, g:i:s A e
    date_format_sidebar_uploaded:            M j, Y, g:i:s A e
    date_format_sidebar_taken_at:            M j, Y, g:i:s A e
    date_format_hero_min_max:                F Y
    date_format_hero_created_at:             M j, Y, g:i:s A T
    date_format_album_thumb:                 M Y
    upload_chunk_size:                       0
    nsfw_banner_blur_backdrop:               0
    search_pagination_limit:                 1000
    search_minimum_length_required:          4
    photo_layout_justified_row_height:       320
    photo_layout_masonry_column_width:       300
    photo_layout_grid_column_width:          250
    photo_layout_square_column_width:        200
    photo_layout_gap:                        12
    display_thumb_album_overlay:             always
    display_thumb_photo_overlay:             hover
    default_album_thumb_aspect_ratio:        1/1
    use_album_compact_header:                0
    login_button_position:                   left
    back_button_enabled:                     0
    back_button_text:                        Return to Home
    current_job_processing_visible:          1
    SA_random_thumbs:                        0
    job_processing_queue_visible:            1
    login_required:                          0
    enable_unsorted:                         1
    enable_starred:                          1
    enable_recent:                           1
    enable_on_this_day:                      1
    login_required_root_only:                1

**Reply by ildyria** [reply]
Are you behind reverse proxy?

I see that you are using git, did you do `npm run build` ?
Can you check that your APP_URL starts with `https`?
Can you set `APP_FORCE_HTTPS=true` ?

**Reply by s22-tech** [reply]
No, I'm not behind a proxy.
I did run `npm run build`.
I updated my .env file after I sent it to you so the URL is correct.

**Reply by ildyria** [reply]
Now can you try `LIVEWIRE_ENABLED=true`. :)
If it still does not work, just revert. It is fine as I am working on rewriting the full front-end so the Livewire stuff does not really matter.

**Reply by s22-tech** [reply]
Nope.  Still doesn't work.  I'll keep it on false.
